presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/BucketShards.java

```diff
@@ -67,7 +67,7 @@ public int hashCode()
     public String toString()
     {
         return toStringHelper(this)
-                .add("bucketNumber", bucketNumber.getAsInt())
+                .add("bucketNumber", bucketNumber)
```

Comment:
This works but results in an ugly string with "OptionalInt" in it. Ideally, `toStringHelper` should know about optionals and elide them. Instead, do it like `ShardInfo`:
```java
.add("bucketNumber", bucketNumber.isPresent() ? bucketNumber.getAsInt() : null)
...
.omitNullValues()